doc/source/conf.py

```diff
@@ -798,3 +798,40 @@ def setup(app):
     app.add_autodocumenter(AccessorMethodDocumenter)
     app.add_autodocumenter(AccessorCallableDocumenter)
     app.add_directive("autosummary", PandasAutosummary)
+
+
+####### This is an ignore list for broken links found in CI run checks for broken-linkcheck.yml
+
+linkcheck_ignore = [r'http://']
+linkcheck_ignore = [r'http://scatterci.github.io/pydata/pandas:\d+/']
```

Comment:
1. You can't set this variable multiple times. You'll need to put multiple entries in a list:

```suggestion
linkcheck_ignore = [
    r'http://',
    r'http://scatterci.github.io/pydata/pandas:\d+/'
    ...
]
```

2. Not sure what you want to achieve with the `:\d+/` at the end of every line. With this, `http://scatterci.github.io/pydata/pandas` will not get matched and in turn ignored.